Javadoc error: '_' used as an identifier in StructureTools

```
biojava-legacy: An error has occurred in JavaDocs report generation:
[ERROR] Exit code: 1 - biojava-legacy/core/src/main/java/org/biojava/bio/proteomics/StructureTools.java:37:
warning: '_' used as an identifier
[ERROR]   private static final AtomicSymbol _;
[ERROR]                                     ^
[ERROR]   (use of '_' as an identifier might not be supported in releases after Java SE 8)
```

--- a/core/src/main/java/org/biojava/bio/proteomics/StructureTools.java
+++ b/core/src/main/java/org/biojava/bio/proteomics/StructureTools.java
@@ -34,7 +34,7 @@ import org.biojava.bio.symbol.FiniteAlphabet;
 public final class StructureTools {
   private static final FiniteAlphabet struct;
 
-  private static final AtomicSymbol _;
+  private static final AtomicSymbol space;
   private static final AtomicSymbol c;
   private static final AtomicSymbol h;
   private static final AtomicSymbol g;
@@ -50,7 +50,7 @@ public final class StructureTools {
 
       SymbolTokenization sTok = struct.getTokenization("token");
 
-      _ = (AtomicSymbol) sTok.parseToken(" ");
+      space = (AtomicSymbol) sTok.parseToken(" ");
       c = (AtomicSymbol) sTok.parseToken("c");
       h = (AtomicSymbol) sTok.parseToken("h");
       g = (AtomicSymbol) sTok.parseToken("g");
@@ -69,7 +69,7 @@ public final class StructureTools {
   }
 
   public AtomicSymbol get_() {
-    return _;
+    return space;
   }
 
   public AtomicSymbol getC() {

